### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by GddgdgMen**

### CODE REVIEW

File: Tasks/difference.js [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-    object_1[attribute_name] = object_1[attribute_name]
-    }{}
-    if (object_1 in object_2) return false
-    for (attribute_name in object_2) {
-    object_1[attribute_name] = object_2[attribute_name]
-    delete object_1[attribute_name]
-    }{}
-    return object_1
+const diff = (object1, object2) => {
+  for (const key in object2) {
```
Comment (tshemsedinov) [inline Tasks/difference.js:5]: Prefer for..of here

File: Tasks/difference.js [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
+    delete object1[key];
+  }
+  return object1;
```
Comment (tshemsedinov) [inline Tasks/difference.js:8]: Don't change incoming parameters

File: Tasks/intersection.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
+    if (name !== "" && object2[name]) {
+        object2[name] = object1[name];
+    } else {
+      delete object1[name];
+    }
+  }
+  return object1;
+};
+const result = intersection({ a: "uno", b: "due" }, { a: "uno", c: "tre" });
+console.log(result);
```
Comment (tshemsedinov) [inline Tasks/intersection.js:16]: Add \n at EOF

**Review by tshemsedinov — COMMENTED** [review]
- Use eslint and prettier
- Don't change incoming parameters

**Comment by GddgdgMen** [comment]
Updated the code

**Pull request closed without merging**